.github/workflows/_comps-workflow.yml

```diff
@@ -70,8 +70,8 @@ jobs:
               cd ./vllm-openvino && git checkout v0.6.1 && git rev-parse HEAD && cd ../
           fi
           if [[ $(grep -c "vllm-gaudi:" ${docker_compose_yml}) != 0 ]]; then
-               git clone https://github.com/HabanaAI/vllm-fork.git vllm-fork
-               cd vllm-fork && git checkout v0.6.4.post2+Gaudi-1.19.0 && cd ../
+              git clone --depth 1 --branch v0.6.4.post2+Gaudi-1.19.0 https://github.com/HabanaAI/vllm-fork.git
+              sed -i 's/triton/triton==3.1.0/g' vllm-fork/requirements-hpu.txt
```

Comment:
This will fail once upstream adds the version.

Something like this would keep working:
`sed -i 's/triton.*$/triton==3.1.0/' vllm-fork/requirements-hpu.txt`

Or if there could be other modules which include `triton`in their name:
`sed -i 's/^triton[^-a-z]*$/triton==3.1.0/' vllm-fork/requirements-hpu.txt`